### PR TITLE
Fixes #14795 - User w/o edit_params permission can save host

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by dLobatog** [comment]
@tbrisker Thanks, can you add a test for it? This is a bit brittle and it's not a commonly tested path so it will likely break again without us noticing if it's not tested.

File: app/views/common_parameters/_parameter.html.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
   </td>
   <td>
     <%= f.check_box(:hidden_value, :class => 'set_hidden_value hide', :checked => f.object.hidden_value?) if can_edit_params? %>
-    <%= link_to_remove_fields('', f) if authorized_via_my_scope("host_editing", "destroy_params") %><%= _('remove') %>
+    <%= link_to_remove_fields(_('remove'), f) if authorized_via_my_scope("host_editing", "destroy_params") %>
```
Comment (tbrisker) [inline app/views/common_parameters/_parameter.html.erb:10]: No sense showing the `remove` text if it can't be removed

**Comment by tbrisker** [comment]
@dLobatog Added test, found another field that wasn't disabled (the `textarea`).

**Comment by dLobatog** [comment]
Merged as c9405b9b573fafcd0a63c81538c4322180e8c9ce, thanks @tbrisker!

**Pull request closed without merging**